In the console, type clear and hit Enter or Return. 

This will clear the console output. 

Run the file once while the tasks are longer. 

Copy and paste the terminal results (when task_duration is 3) here.

PS C:\Users\15738\Documents\streaming-02-multiple-processes> & C:/Users/15738/AppData/Local/Programs/Python/Python310/python.exe c:/Users/15738/Documents/streaming-02-multiple-processes/streaming-02-multiple-processes/multiple_processes.py


STARTING UP.............................
It's 2023-01-19 at 09:38 PM

This file is running on:    nt Windows 10
The Python version is:      3.10.7

The Python interpreter is at:
 C:\Users\15738\AppData\Local\Programs\Python\Python310\python.exe


DELETED tabble pets.
CREATED table pets.

P1 getting ready to insert Ace the Dog.
P1 ADDED Ace the Dog.
P2 getting ready to insert Cooper the Rabbit.
P3 getting ready to insert Emma the Rabbit.
P1 getting ready to insert Buddy the Dog.
P1 ADDED Buddy the Dog.
P3 ADDED Emma the Rabbit.
Process Process-2:
Traceback (most recent call last):
  File "C:\Users\15738\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 314, in _bootstrap
    self.run()
  File "C:\Users\15738\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "c:\Users\15738\Documents\streaming-02-multiple-processes\streaming-02-multiple-processes\multiple_processes.py", line 106, in process_two
    insert_pet("P2", "Cooper", "Rabbit")
  File "c:\Users\15738\Documents\streaming-02-multiple-processes\streaming-02-multiple-processes\multiple_processes.py", line 95, in insert_pet
    cur.execute(sql)
sqlite3.OperationalError: database is locked
P3 getting ready to insert Felix the Cat.
P3 ADDED Felix the Cat.
PS C:\Users\15738\Documents\streaming-02-multiple-processes> 